modules/dag-shared/src/test/scala/org/tessellation/dag/domain/block/generators.scala

```diff
@@ -0,0 +1,26 @@
+package org.tessellation.dag.domain.block
+
+import cats.data.{NonEmptyList, NonEmptySet}
+
+import org.tessellation.dag.domain.block.{BlockReference, DAGBlock}
+import org.tessellation.schema.generators._
+import org.tessellation.schema.height.Height
+import org.tessellation.security.hash.ProofsHash
+
+import org.scalacheck.{Arbitrary, Gen}
+
+object generators {
+
+  val blockReferenceGen: Gen[BlockReference] =
```

Comment:
Block Reference already has an `Arbitrary` instance (which can be mapped to `Gen` if necessary by calling `arbitrary` method).